scripts/findChrome.js

```diff
@@ -197,6 +198,24 @@ async function downloadChromium() {
   }
 }
 
+function isSuitableVersion(executablePath) {
+  let versionOutput
+  // in case installed Chrome is not runnable
+  try {
+    versionOutput = execSync(`${executablePath} --version`).toString()
+  } catch (e) {
+    return false
+  }
+  const versionRe = /(Google Chrome|Chromium) ([0-9]{2}).*/
+
+  const match = versionOutput.match(versionRe)
+  if (match && match[2]) {
+    const version = +match[2]
+    return version >= 75
```

Comment:
We need to move it to `const` in the beginning of the file

Reply:
Will it be right to add a non-const variable to the const block? Since we don't know local version at start.

Reply:
I am talking about

```js
const MIN_CHROME_VERSION = 75
```

We will change this version requirement tin the future